modules/luci-base/luasrc/sys.lua

```diff
@@ -189,6 +189,23 @@ local function _nethints(what, callback)
 			end
 		end
 	)
+	
+	cur:foreach("dhcp", "odhcpd",
+		function(s)
+			if s.leasefile and fs.access(s.leasefile) then
+				for e in io.lines(s.leasefile) do
+					local iface, duid, iaid, name, ts, id, length, ip = e:match("^# (%S+) (%S+) (%S+) (%S+) (-?%d+) (%S+) (%S+) (.*)")
+					if ip and iaid == "ipv4" then
+						ip = ip:gsub("/[0-9]+","")
+						mac = luci.ip.checkmac(duid:gsub("^(%x%x)(%x%x)(%x%x)(%x%x)(%x%x)(%x%x)$", "%1:%2:%3:%4:%5:%6")) or "00:00:00:00:00:00"
```

Comment:
If the mac cannot be parsed, skip the entry.
Also: `local mac`

Reply:
mac is already local